extensions/cl_intel_unified_shared_memory.asciidoc

```diff
@@ -231,12 +231,13 @@ Optional allocation properties may also be queried using *clGetMemAllocInfoINTEL
 ----
 typedef cl_uint cl_mem_info_intel;
 
-#define CL_MEM_ALLOC_TYPE_INTEL         0x419A
-#define CL_MEM_ALLOC_BASE_PTR_INTEL     0x419B
-#define CL_MEM_ALLOC_SIZE_INTEL         0x419C
-#define CL_MEM_ALLOC_DEVICE_INTEL       0x419D
+#define CL_MEM_ALLOC_TYPE_INTEL               0x419A
+#define CL_MEM_ALLOC_BASE_PTR_INTEL           0x419B
+#define CL_MEM_ALLOC_SIZE_INTEL               0x419C
+#define CL_MEM_ALLOC_DEVICE_INTEL             0x419D
+#define CL_MEM_ALLOC_BUFFER_LOCATION_INTEL    0x419E
```

Comment:
Could you add some spec wording about the usage of `CL_MEM_ALLOC_BUFFER_LOCATION_INTEL`  please

Additionally, your example
```c
cl_mem_properties_intel good_property[3] = {
CL_MEM_ALLOC_BUFFER_LOCATION_INTEL, 0,
0};
```

Is using `CL_MEM_ALLOC_BUFFER_LOCATION_INTEL` as a property but the above enums are for `clGetMemAllocInfoINTEL` so I think this new enum should be split off rather than grouped with them

Reply:
Thank you for reviewing and providing suggestions!

I have added the spec in `cl_intel_mem_alloc_buffer_location.asciidoc`

Regarding CL_MEM_ALLOC_BUFFER_LOCATION_INTEL shouldn't be `CL_MEM_ALLOC...` group: This enum will be used in the clGetMemAllocInfoINTEL for user to check whether the place they are allocating is correct. Sorry I forgot to mention this in the commit message, I now have added it to the spec.

Reply:
Thanks for the response,

> I have added the spec in cl_intel_mem_alloc_buffer_location.asciidoc

Creating a new document `cl_intel_mem_alloc_buffer_location.asciidoc` implies that this is a new extension that depends on `cl_intel_unified_shared_memory`, is this the intention?

Reply:
Sorry I did not know that is the implication of a new doc. That is indeed no the intention. I have moved the info from the other doc to current doc. Thanks for letting me know!